README.md

```diff
@@ -1,2 +1,101 @@
-# docs
-documentation
+# Using of PAX.express
+
+The PAX.express service is used for the delivery of firmware to esp32 paxcounter devices. This is done by using the OTA feature of the [ESP32-paxcounter project](https://github.com/cyberman54/ESP32-Paxcounter). Our fork can be found [here](https://github.com/paxexpress/ESP32-Paxcounter), which is already setup to use PAX.express to deliver firmware updates. The PAX.express cli can be found [here](https://github.com/paxexpress/paxexpress-cli).
+
+## Setup guide
+### Tooling 
+
+Please clone the [paxexpress-cli](https://github.com/paxexpress/paxexpress-cli) to your filesystem. 
+For installation informations read the [README.md](https://github.com/paxexpress/paxexpress-cli/blob/master/README.md).
+
+### Registration and login
+It's a good idea to switch to the poetry shell now:
```

Comment:
If a precompiled binary is used, the reader might never have heard of poetry and is lost at the point